addon/components/frost-radio-button.js

```diff
@@ -39,57 +40,82 @@ export default Component.extend(PropTypeMixin, {
   },
 
   // == Computed properties  ===================================================
-
-  groupId: readOnly('parentView.id'),
-  groupValue: readOnly('parentView.value'),
-  onChange: readOnly('parentView.onChange'),
-
-  checked: computed('groupValue', 'value', function () {
-    return this.get('groupValue') === this.get('value')
-  }),
-
-  tabindex: Ember.computed('disabled', function () {
-    return this.get('disabled') ? -1 : 0
-  }),
+  @computed('parentView.id')
+  groupId: {
+    get (id) {
+      return id
+    },
+    set (value) {
+      set(this, 'parentView.id', value)
+    }
+  },
+  @computed('parentView.value')
+  groupValue: {
+    get (value) {
+      return value
+    },
+    set (value) {
+      set(this, 'parentView.value', value)
+    }
+  },
+  @computed('parentView.onChange')
+  onChange: {
+    get (onChange) {
+      return onChange
+    },
+    set (value) {
+      set(this, 'parentView.onChange', value)
+    }
+  },
+  @computed('parentView.value', 'value')
+  checked (groupValue, value) {
+    return groupValue === value
+  },
+  @computed('disabled')
+  tabindex (disabled) {
+    return disabled ? -1 : 0
+  },
 
   // == Functions ===============================================================
 
   _createEvent (_event, _target) {
-    let event = Ember.$.Event(null, _event)
+    let event = Ember.$.Event(this, null, _event)
     let target = Ember.$.clone(_target)
-    target.id = this.get('groupId')
+    target.id = get(this, 'groupId')
     event.target = target
     return event
   },
 
   // == Events ===============================================================
 
-  init () {
+  didInsertElement () {
     this._super(...arguments)
     Ember.assert(
-      `${this.toString()} must be initialized in the yield block of 'frost-radio-group'`,
-      /frost-radio-group/.test(this.parentView.toString()))
+      `${this.toString()} must be initialized inside of 'frost-radio-group'`,
+      /frost-radio-group/.test(get(this, 'parentView').toString()))
     Ember.assert(
       `${this.toString()} must be initialized with a 'value' property`,
-      this.get('value')
+      get(this, 'value')
     )
   },
-
+  // fix for Ember 2.8 on application destroy
+  willDestroyElement () {
+    set(this, 'parentView', null)
```

Comment:
`afterEach` hook from the `destroy-app` helper was throwing an error in 2.8. This addresses that.